### DISCUSSION TITLE
Host server on different computer on the same network

### DISCUSSION THREAD

**Discussion by jalxp** [ptrumpis/snap-camera-server#137] [Q&A] [answered]
I'm trying to host the server on a computer and use another one as a client. How can I achieve this? They are both on the same network. 
Is it enough to run docker on machine A and edit the hosts file on machine B so that it knows the address of machine A?

**Comment by ptrumpis** [comment, marked as answer]
Yes, you need to edit the hosts file of machine B and also import/trust the SSL certificate on machine B

**Reply by jalxp** [reply]
Hey @ptrumpis, thanks for the reply! Worked like a charm, had totally forgot the certs.
Cheers!
